### PR TITLE
Updated markup for community page

### LINKED ISSUE #432: [redesign] Community Page - Update Markup
> If this is your first time tackling an issue that is part of the redesign project please read all the **Notes** section at the bottom of this issue before claiming it and starting to work on it

## To Do: 
- [ ] Update the markup for the `/community` page

--- 

### Notes (important) 

This issue is part of the [Ember Website Redesign RFC](https://github.com/emberjs/rfcs/blob/master/text/0425-website-redesign.md) and is one of the **last remaining issues** before we are ready to release. As we are moving very quickly on this project you should comment on this issue that you are taking it on so that we do not have multiple people working on the same issue. **Note:** please don't take on an issue if you do not think you can complete it within 24 hours 🙏 

This issue is **strictly to only update the markup** of a page on the website and should be possible to do using only components and class helpers in the new [ember-styleguide](https://deploy-preview-145--ember-styleguide.netlify.com/).

If you find that you need to create page-specific styles, complete the page as best you can and open a PR **without** adding any styles. Make a note in your PR what was missing from the styleguide that prevented you from completing the page.

When creating your PR you should target the `redesign-rfc` branch. This way when your PR is merged it will not update `master` it will instead update the long-lived branch that is tracking the website redesign work.

### PR DESCRIPTION
**Pull request by mswanson**
fixes #432 

There was no style for floated images so I used the `layout-grid` class and put them in a 2 col space with the text in a 4 col space beside them. Some of the images are not high res enough and look a bit pixelated.

Also I bumped the video res from 640x360 to 854x480 to make it fit better under the 2 col layout above it.

### CODE REVIEW

File: app/templates/community/index.hbs [hunk 1]
```diff
@@ -1,122 +1,127 @@
...
-        src="https://www.youtube.com/embed/rY5D38RQoEg?rel=0"
-        frameborder="0"
-        allowfullscreen
-      >
-      </iframe>
+  </section>
+  <section class="margin-top-medium" aria-labelledby="section-reporting-bugs">
+    <div class="layout-grid">
+      <div class="col-4-large">
+        <h2>Something Fishy? Report it to the Team</h2>
```
Comment (MelSumner) [inline app/templates/community/index.hbs]: this is missing the id "section-reporting-bugs"

File: app/templates/community/index.hbs [hunk 2]
```diff
@@ -1,122 +1,127 @@
...
+      </div>
+    </div>
+  </section>
+  <section class="margin-top-medium" aria-labelledby="section-contribute-to-ember">
+    <div class="layout-grid">
+      <div class="col-2-large">
+        <img alt="Contribute" class="img-fluid" src="/images/community/give.png">
+      </div>
+      <div class="col-4-large">
+        <h2 class="text-center">In a Giving Mood? Contribute to the Project</h2>
```
Comment (MelSumner) [inline app/templates/community/index.hbs]: this is missing the id "section-contribute-to-ember"

File: app/templates/community/index.hbs [hunk 3]
```diff
@@ -1,122 +1,127 @@
...
+          create a fork of the project on GitHub and submit a pull request. Please be sure to include unit tests and
+          documentation for any new features you add. See the <a
+            href="https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md">full guidelines</a> for more information.
+        </p>
+      </div>
     </div>
+  </section>
+  <div class="layout-grid">
+    <section class="margin-top-medium col-3-large text-center" aria-labelledby="section-meetups">
+      <h2>Feeling Lonely? Join a Meetup</h2>
```
Comment (MelSumner) [inline app/templates/community/index.hbs]: this is missing the id "section-meetups"

File: app/templates/community/index.hbs [hunk 4]
```diff
@@ -1,122 +1,127 @@
...
+      </p>
+      <p>
+        {{#link-to "community.meetups" class="es-button"}}
+          Find a meetup
+        {{/link-to}}
+      </p>
+    </section>
+
+    <section class="margin-top-medium col-3-large text-center" aria-labelledby="section-mascots">
+      <h2>Met The Mascots? You'll Love Them!</h2>
```
Comment (MelSumner) [inline app/templates/community/index.hbs]: this is missing the id "section-mascots"

File: app/templates/community/index.hbs [hunk 5]
```diff
@@ -1,122 +1,127 @@
...
-      alt="Contribute"
-      class="section-image pull-left"
-      src="/images/community/give.png"
-    >
+<div class="container">
+  <h1>Get Involved: Join the Growing Ember.js Community</h1>
+  <section class="margin-top-large" aria-labelledby="section-community">
+    <div class="layout-grid">
+      <div class="col-2-large">
+        <img alt="SOS" class="img-fluid" src="/images/community/sos.png">
```
Comment (MelSumner) [inline app/templates/community/index.hbs]: this should have a blank alt tag and `role="presentation"`

File: app/templates/community/index.hbs [hunk 6]
```diff
@@ -1,122 +1,127 @@
...
+      </div>
+      <div class="col-4-large">
+        <h2 class="margin-bottom-small">Stuck? Lost? Get Help from the Community</h2>
```
Comment (MelSumner) [inline app/templates/community/index.hbs]: this is missing the id "section-community"

File: app/templates/community/index.hbs [hunk 7]
```diff
@@ -1,122 +1,127 @@
...
+          one.
+        </p>
+        <p>
+          Please note that these resources are not managed or moderated by the Ember Core Team. They are public forums.
+        </p>
+      </div>
+    </div>
+  </section>
+  <section class="margin-top-medium" aria-labelledby="section-latest-news">
+    <h2>Stay Up to Date with the Latest News</h2>
```
Comment (MelSumner) [inline app/templates/community/index.hbs]: this is missing  `id="section-latest-news"`

File: app/templates/community/index.hbs (hunk 1 above)
Comment (mswanson) [reply app/templates/community/index.hbs]: ok I will update this and the PR and #455 to make this change

File: app/templates/community/index.hbs [hunk 8]
```diff
@@ -37,86 +34,69 @@
...
+        Tomster and Zoey have been crafted with love, and are constantly getting makeovers. Events, User Groups, Countries&mdash;sky's the limit!
+      </p>
+      {{#link-to "mascots" class="es-button"}}
+        Meet the Mascots
+      {{/link-to}}
+    </section>
   </div>
-</section>
+  <section class="margin-top-large text-center" aria-labelledby="section-meetup-video">
+    <iframe width="854" height="480" src="https://www.youtube.com/embed/rY5D38RQoEg?rel=0" frameborder="0" allowfullscreen></iframe>
```
Comment (mansona) [inline app/templates/community/index.hbs]: So the iframe needs `max-width: 100%` on it to make sure that it doesn't bust out of the container on a mobile view 🤔 

I'm personally ok with this being just in a `style=""` attribute but what do other people think?

Comment (mswanson) [reply app/templates/community/index.hbs]: We could throw it inside a grid layout then the 100% max width would always be relative to the wrapper, allowing the grid logic to determine the width on mobile.

Comment (mswanson) [reply app/templates/community/index.hbs]: here is some utility css I have used in the past to set aspect ratio and still allow for dynamic resizing

```
/**
 * Aspect Ratios
**/

/* structure */
.ar-parent {
  position: relative;
  height: 0;
}
.ar-child {
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  z-index: 100;
  height: 100%;
  width: 100%;
}

/* size options added to the wrapper */
.ratio-square {
  padding-bottom: 100%;
}
.ratio-16x9 {
  padding-bottom: 56.25%;
}
.ratio-9x16 {
  padding-bottom: 177.77%;
}
.ratio-4x3 {
  padding-bottom: 75%;
}
.ratio-3x4 {
  padding-bottom: 133.33%;
}
.ratio-6x4 {
  padding-bottom: 66.6%;
}
.ratio-4x6 {
  padding-bottom: 150%;
}
.ratio-8x5 {
  padding-bottom: 62.5%;
}
.ratio-5x8 {
  padding-bottom: 160%;
}
.ratio-7x5 {
  padding-bottom: 71.42%;
}
.ratio-5x7 {
  padding-bottom: 140%;
}
```

Comment (mswanson) [reply app/templates/community/index.hbs]: here it is locally on mobile 
Screenshot 2019-10-30 17 44 00

Comment (mansona) [reply app/templates/community/index.hbs]: Yes I would be happy with using the exiting grid layout here if that solves the issue 🎉 

You might need to make use of the grid helpers that aren't properly documented yet (see https://github.com/ember-learn/ember-styleguide/issues/235) if you want it to be 100% in both desktop and mobile 

`class="col-6-large col-4"`

And thanks for sticking with this implementation, it turns out that it was more complex then I was expecting 🙈

Comment (mswanson) [reply app/templates/community/index.hbs]: Hey @mansona sorry I'm just getting back to this... So to be clear in order to get the screenshot above I had to add the utility css in the [previous comment](https://github.com/ember-learn/ember-website/pull/456#discussion_r340889804). 

I assume that if you are ok with adding that CSS that it needs to go in the Ember Styleguide. I am happy to post a PR for to that repo with the CSS or whatever you think is best. Let me know how to proceed.

Comment (mansona) [reply app/templates/community/index.hbs]: So we have talked about this at length (twice!) and decided that we're going to add one helper to fix the aspect ratios of videos. You can see more info here: https://github.com/ember-learn/ember-styleguide/issues/244 🎉

**Pull request merged**